src/containers/Application/Dashboard.js

```diff
@@ -63,15 +70,21 @@ const ApplicationDashboardContainer = () => {
   }, [setLivesiteDoc])
 
```

Comment:
instead of handling loading in the child components, I think you can do something like this and remove the loading prop that is passed down. 
```
  if (isLoadingAppStatus) return <Spinner />
```